test/exports.spec.js

```diff
@@ -1,15 +1,14 @@
-import * as chai from 'chai'
-import * as path from 'path';
-import * as glob from 'glob';
-import toCamelCase from 'camelcase';
-import interopRequire from 'interop-require';
-import * as decorators from '../';
+const chai = require('chai');
+const path = require('path');
+const glob = require('glob');
+const camelCase = require('camelCase');
+const interopRequire = require('interop-require');
+const decorators = require('core-decorators');
```

Comment:
Oh yes..I forgot they didn't support it. I say we punt.

Re TS updating to stage-3: I haven't heard, but I would bet money TS is waiting for the decorators proposal to be finalized stage-4 because in its current stage-3 form it's going to be quite the pain in the ass of a breaking change for the TS community aka mostly Angular people, so they want to be 100% sure of the changes so they only have to do it once.